packages/front-end/components/Features/FeatureImplementationModal.tsx

```diff
@@ -0,0 +1,206 @@
+    },
+
+    {
+      id: "go",
+      display: "Go",
+      language: "go",
+      boolean: `if gb.Feature(${JSON.stringify(feature.id)}).On {
+  // serve the feature
+}`,
+      value: `color := gb.Feature(${JSON.stringify(
+        feature.id
+      )}).GetValueWithDefault("blue")
+fmt.Println(color)`,
+      docs: "https://docs.growthbook.io/lib/go",
+    },
+
+    {
+      id: "kotlin",
+      display: "Kotlin (Android)",
+      language: "kotlin",
+      boolean: `if (gb.feature(${JSON.stringify(feature.id)}).on) {
+  // Feature is enabled!
+}`,
+      value: `val feature = gb.feature(${JSON.stringify(feature.id)})
+println(feature.value)
+`,
+      docs: "https://docs.growthbook.io/lib/kotlin",
+    },
+
+    {
+      id: "php",
+      display: "PHP",
+      language: "php",
+      boolean: `if ($growthbook->isOn(${JSON.stringify(feature.id)})) {
+  echo "It's on!";
+} else {
+  echo "It's off :(";
+}`,
+      value: `$color = $growthbook->getValue(${JSON.stringify(
+        feature.id
+      )}, "blue");
+
+echo "<button style="color:\${color}">Click Me!</button>";`,
+      docs: "https://docs.growthbook.io/lib/php",
+    },
+
+    {
+      id: "python",
+      display: "Python",
+      language: "python",
+      boolean: `if gb.isOn(${JSON.stringify(feature.id)}):
+  print("My feature is on!")`,
+      value: `color = gb.getFeatureValue(${JSON.stringify(
+        feature.id
+      )}, "blue")`,
+      docs: "https://docs.growthbook.io/lib/python",
+    },
+
+    // ruby: {
+    //   python: ``,
+    //   boolean: ``,
+    //   value: ``,
+    //   docs: "https://docs.growthbook.io/lib/ruby",
+    // },
+  ];
+
+  const codeType = feature.valueType === "boolean" ? "boolean" : "value";
+
+  if (fullSnippet) {
+    return (
+      <CodeSnippetModal
+        close={close}
+        featureId={feature.id}
+        defaultLanguage={language}
+      />
+    );
+  }
+
+  return (
+    <Modal
+      open={true}
+      close={close}
+      size="lg"
+      closeCta="Close"
+      header="Feature implementation"
+    >
+      <p>
+        {first && <>Congratulations on adding your first feature! </>}
+        Here is the example code on how to add it to your project
+      </p>
+      <div>
+        <ControlledTabs
+          active={language}
+          setActive={(language) => setLanguage(language as Language)}
+        >
+          {codeSamples.map((o, i) => {
+            return (
+              <Tab key={i} display={o.display} id={o.id}>
+                <p>
+                  Read the{" "}
+                  <a href={o.docs} target="_blank" rel="noopener noreferrer">
+                    {o.id} SDK docs
+                  </a>
+                  or view the{" "}
+                  <a
+                    href="#"
+                    onClick={(e) => {
+                      e.preventDefault();
+                      setFullSnippet(true);
+                    }}
+                  >
+                    full implementation guide
```

Comment:
Calling this a guide makes me think it will take me to a tutorial instead of just another code snippet. Maybe "full implementation example" instead?